app/views/homerooms/show.html.erb

```diff
@@ -74,19 +74,22 @@
             <th class="risk sort-default" data-sort-method="reverse_number">
               <span class="table-header">Risk</span>
             </th>
-            <th class="program">
+            <!-- The data-sort-method attribute forces Tablesort to use a specific sorting method, instead of the default sort it uses. You can see the method at: app/assets/javascripts/roster/sorts/prog_assign_sort.js -->
```

Comment:
Thanks for the correction. Just fixed it and pushed.
Note the there are previous comments in that page, written in HTML format. 
Shall I change them as well?